desktop/main.dev.ts

```diff
@@ -68,8 +68,31 @@ StoreService.init();
 
 // State
 const context = getDefaultAppContext();
-// App behaviors
-new AutoStartManager(); // eslint-disable-line no-new
+
+const IS_AUTO_START_ENABLED = 'isAutoStartEnabled';
+
+if (StoreService.get(IS_AUTO_START_ENABLED)) {
+  console.log('init', StoreService.get(IS_AUTO_START_ENABLED));
```

Comment:
Are these console.logs for debugging/testing purposes and you will revert this commit? :)